### PR TITLE
Update community guidelines page

### PR DESCRIPTION
**Pull request by nick-mon1**
### Summary

Update copy and added `do's & don'ts` table.
This code will be refactored into a component for re-use. Currently it is used as a one-off.

https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-community-guidelines/communities/community-guidelines/

**Before**
Screen Shot 2023-03-10 at 12 15 28-fullpage

**After**
Screen Shot 2023-03-10 at 12 15 40-fullpage

Screen Shot 2023-03-10 at 12 18 52-fullpage

### CODE REVIEW

File: themes/digital.gov/layouts/partials/core/community-guidelines.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
+          <use xlink:href="{{ "uswds/img/sprite.svg#cancel" | relURL }}"></use>
+        </svg>
+        What you shouldn't do
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr scope="row">
+      <td>
```
Comment (mejiaj) [inline themes/digital.gov/layouts/partials/core/community-guidelines.html:21]: Were the stacked data attributes left out intentionally?

Notice the `data-label` on `<th>` and `<td>`.
image

https://designsystem.digital.gov/components/table/#responsive-stacked-table

**Review by mejiaj — COMMENTED** [review]
@nick-mon1 table looks good, just some comments on things we can improve. I realize there's an urgency for this so we can iterate for next sprint based on what's feasible now.

File: data/images/screenshot-1.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+  # https://s3.amazonaws.com/digitalgov/screenshot-1.png
+  # Image shortcode: {{< img src=screenshot-1 >}}'
+  date     :  2023-03-08 09:39:49 -0400
+  uid      :  screenshot-1
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply data/images/screenshot-1.yml:5]: This file shouldn't be here, it is now removed.

File: data/images/screenshot-2.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+  # https://s3.amazonaws.com/digitalgov/screenshot-2.png
+  # Image shortcode: {{< img src=screenshot-2 >}}'
+  date     :  2023-03-08 09:53:52 -0400
+  uid      :  screenshot-2
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply data/images/screenshot-2.yml:5]: This file shouldn't be here, it is now removed.

File: data/images/screenshot-1.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  width    :  2560
+  height   :  1466
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply data/images/screenshot-1.yml:7]: This file shouldn't be here, it is now removed.

File: themes/digital.gov/src/scss/new/_uswds-overrides.scss [hunk 5]
```diff
@@ -16,10 +16,34 @@
...
+    @include u-color("red-50v");
+  }
+
+  tbody tr td {
+    width: 50%;
+  }
+
+  li {
+    list-style-position: inside;
+  }
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/_uswds-overrides.scss:44]: Removed!

File: themes/digital.gov/layouts/partials/core/community-guidelines.html (hunk 1 above)
Comment (mejiaj) [reply themes/digital.gov/layouts/partials/core/community-guidelines.html]: Mobile view looks even better!

image

**Review by mejiaj — DISMISSED** [review]

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**